cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -577,6 +580,8 @@ class HydratorPlusPlusConfigStore {
       this.state.config.schedule = this.state.config.schedule || this.HYDRATOR_DEFAULT_VALUES.schedule;
     } else if (artifact.name === this.GLOBALS.etlRealtime) {
       this.state.config.instances = this.state.config.instances || this.HYDRATOR_DEFAULT_VALUES.instance;
+    } else if (artifact.name === this.GLOBALS.eltSqlPipeline) {
+      this.state.config.schedule = this.state.config.schedule || this.HYDRATOR_DEFAULT_VALUES.schedule;
```

Comment:
isn't this already set by line 580?

Reply:
Not applicable, sql-pipeline is not a batch pipeline